assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -134,13 +135,15 @@ class PageEdit extends Component {
 			mediaType = media.type;
 		}
 
-		const mediaUrl = has( media, [ 'sizes', MAX_IMAGE_SIZE_SLUG, 'url' ] ) ? media.sizes[ MAX_IMAGE_SIZE_SLUG ].url : media.url;
```

Comment:
@kienstra This appears no longer to be necessary when the `srcset` is present. 

Originated in https://github.com/ampproject/amp-wp/pull/2509/commits/0bd76bc652288cb82448d6134448323d5d853778

Reply:
Ah, that makes sense.